cli/replicasets/weight.go

```diff
@@ -0,0 +1,72 @@
+package replicasets
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/apex/log"
+	"github.com/tarantool/cartridge-cli/cli/context"
+)
+
+func SetWeight(ctx *context.Ctx, args []string) error {
+	if ctx.Replicasets.ReplicasetName == "" {
+		return fmt.Errorf("Please, specify replicaset name via --replicaset flag")
+	}
+
+	if err := FillCtx(ctx); err != nil {
+		return err
+	}
+
+	if len(args) != 1 {
+		return fmt.Errorf("Should be specified one argument - replicaset weight")
+	}
+
+	weight, err := strconv.ParseFloat(args[0], 64)
+	if err != nil {
+		return fmt.Errorf("Failed to parse specified weight. Please, specify valid float")
+	}
+
+	instancesConf, err := getInstancesConf(ctx)
+	if err != nil {
+		return fmt.Errorf("Failed to get instances configuration: %s", err)
+	}
+
+	conn, err := getControlConn(instancesConf, ctx, nil)
+	if err != nil {
+		return fmt.Errorf("Failed to connect to Tarantool instance: %s", err)
```

Comment:
I'm tired to see such errors. Consider moving connection to tarantool directly to command context. You use it anywhere.

Reply:
I've added `connectToSomeRunningInstance` and `connectToSomeJoinedInstance` functions with comments.